### PR TITLE
ci: add drone.io for arm64 verification

### PR DESCRIPTION
**Pull request by JunHe77**
Change-Id: Ib357be80e7e9d7c0ab62433ee5fda4b962592553
Signed-off-by: Jun He <jun.he@arm.com>

### CODE REVIEW

**Comment by JunHe77** [comment]
Hi, @gbtucker, to enable these new jobs Drone.io needs to be allowed to access repo just as travis does.

**Comment by gbtucker** [comment]
Thanks, I'll look into granting drone.io.

**Comment by JunHe77** [comment]
Ping @gbtucker , not sure if any help is needed. If yes, maybe @mrtomparker can help. 😃

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- name: arm64-linux-extended-tests
+  image: ubuntu:xenial
+  environment:
+    TEST_TYPE: ext
+  commands:
+  - if [ -n "$C_COMPILER" ]; then export CC="$C_COMPILER"; fi
+  - if [ -n "$AS_ASSEMBL" ]; then export AS="$AS_ASSEMBL"; fi
+  - apt-get -qq update
+  - apt-get install -qq -y build-essential git indent libtool libz-dev software-properties-common yasm autoconf
+  - apt-get install -qq -y binutils-mingw-w64-x86-64 gcc-mingw-w64-x86-64
```
Comment (gbtucker) [inline .drone.yml]: Not sure why you would want to install tools for cross x86-64 windows here.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 Intel(R) Intelligent Storage Acceleration Library
 =================================================
 
-[![Build Status](https://travis-ci.org/intel/isa-l.svg?branch=master)](https://travis-ci.org/intel/isa-l)
+x86_64: [![Build Status](https://travis-ci.org/intel/isa-l.svg?branch=master)](https://travis-ci.org/intel/isa-l)
+arm64: [![Build Status](https://cloud.drone.io/api/badges/intel/isa-l/status.svg?branch=master)](https://cloud.drone.io/intel/isa-l)
```
Comment (gbtucker) [inline README.md]: Let's remove this until we have a history of it passing.

File: .drone.yml (hunk 1 above)
Comment (JunHe77) [reply .drone.yml]: Thanks for pointing this out. Has removed this.

**Comment by gbtucker** [comment]
Integrated

**Pull request closed without merging**